test/functional/aws_standalone_vpc.go

```diff
@@ -0,0 +1,582 @@
+	clusterTag, err := getClusterID(ctx, testingCtx.Client)
+	if err != nil {
+		t.Fatal("could not get cluster id", err)
+	}
+
+	// verify vpc
+	vpc, err := verifyVpc(ec2Sess, clusterTag, expectedCidr)
+	testErrors.vpcError = err.(*networkConfigTestError).vpcError
+
+	// fail immediately if the vpc is nil, since all of the
+	// other network components are associated with it
+	if vpc == nil {
+		t.Fatal(testErrors.Error())
+	}
+
+	// verify subnets
+	subnets, err := verifySubnets(ec2Sess, clusterTag, expectedCidr)
+	testErrors.subnetsError = err.(*networkConfigTestError).subnetsError
+
+	// verify security groups
+	err = verifySecurityGroup(ec2Sess, clusterTag)
+	testErrors.securityGroupError = err.(*networkConfigTestError).securityGroupError
+
+	// we have to manually construct the subnet group names for rds and elasticache,
+	// since tag filtering isnt currently available
+	name := resources.ShortenString(fmt.Sprintf("%s-%s", clusterTag, "subnet-group"), 40)
+
+	// build array list of all vpc private subnets
+	var subnetIDs []*string
+	for _, subnet := range subnets {
+		subnetIDs = append(subnetIDs, subnet.SubnetId)
+	}
+
+	// verify rds subnet groups
+	rdsSvc := rds.New(session)
+	err = verifyRdsSubnetGroups(rdsSvc, name, subnetIDs)
+	testErrors.rdsSubnetGroupsError = err.(*networkConfigTestError).rdsSubnetGroupsError
+
+	// verify elasticache subnet groups
+	cacheSvc := elasticache.New(session)
+	err = verifyCacheSubnetGroups(cacheSvc, name, subnetIDs)
+	testErrors.cacheSubnetGroupsError = err.(*networkConfigTestError).cacheSubnetGroupsError
+
+	// verify peering connection
+	conn, err := verifyPeeringConnection(ec2Sess, clusterTag, expectedCidr, aws.StringValue(vpc.VpcId))
+	testErrors.peeringConnError = err.(*networkConfigTestError).peeringConnError
+
+	// verify standalone vpc route table
+	err = verifyStandaloneRouteTable(ec2Sess, clusterTag, conn)
+	testErrors.standaloneRouteTableError = err.(*networkConfigTestError).standaloneRouteTableError
+
+	// verify cluster route table
+	err = verifyClusterRouteTables(ec2Sess, clusterTag, expectedCidr, conn)
+	testErrors.clusterRouteTablesError = err.(*networkConfigTestError).clusterRouteTablesError
+
+	// if any error was found, fail the test
+	if testErrors.isValid() {
+		t.Fatal(testErrors.Error())
+	}
+}
+
+// verify that the standalone vpc is created
+func verifyVpc(session *ec2.EC2, clusterTag, expectedCidr string) (*ec2.Vpc, error) {
+	newErr := &networkConfigTestError{
+		vpcError: []error{},
+	}
+
+	// filter vpcs by integreatly cluster id tag
+	describeVpcs, err := session.DescribeVpcs(&ec2.DescribeVpcsInput{
+		Filters: []*ec2.Filter{
+			{
+				Name:   aws.String("tag:integreatly.org/clusterID"),
+				Values: []*string{aws.String(clusterTag)},
+			},
+		},
+	})
+	if err != nil {
+		newErr.vpcError = append(newErr.vpcError, fmt.Errorf("could not find vpc: %v", err))
+		return nil, newErr
+	}
+
+	// only one vpc is expected
+	vpcs := describeVpcs.Vpcs
+	if len(vpcs) != 1 {
```

Comment:
Will there be always only 1 vpc present, even if we have multiple clusters in the same AWS account?

Reply:
It should be, since we're filtering by the cluster tag here - https://github.com/integr8ly/integreatly-operator/pull/937/files/850cece6bbca39fe59c8eea1d0be9ec7fdf5d90d#diff-d235060cf6f899b2b7562835f9853f33R230